src/stream/share.native.ts

```diff
@@ -0,0 +1,18 @@
+
+import * as Etebase from "etebase";
+import * as Sharing from "expo-sharing";
+import { cacheToFile, getFilename, getItemData } from "./utils";
+
+export function canShare() {
+  return true;
+}
+
+export async function shareItem(item: Etebase.Item) {
+  const { name, content } = await getItemData(item);
+  const filename = getFilename(name);
+  const uri = await cacheToFile(filename, content);
+  await Sharing.shareAsync(uri, {
+    mimeType: "text/plain",
```

Comment:
I think it should be `text/markdown`, and the second line adjusted accordingly too.

Reply:
I used `text/plain` because any app that can read plain text can read a markdown file. It doesn't need to be able to read markdown files specifically. It would diminish the number of apps we can share to.

Reply:
Are you sure? Apps that support generic text should just support `text/*`. Apps that only support markdown will only support `text/markdown` and won't open `text/plain`.

Reply:
I've tested it and for example Markor doesn't show up with `text/markdown`.

Reply:
That's a bug in markor then... I don't know what to tell you though. Maybe text/plain is the safer bet. I just hate being part of the problem.

Reply:
I made a mistake, when we share `text/*` data we're supposed to send the content, not the file. So that means either use React Native's `Share` or React Native Share, and both just send it as `plain/text`…